packages/uswds-core/src/styles/mixins/layout-grid/_grid-container.scss

```diff
@@ -13,7 +14,11 @@
     $props: list.nth($props, 1);
     $margin-x: append-important($props, auto);
     @include u-margin-x($margin-x);
-    @include u-maxw($props);
+    @if map.has-key($custom-breakpoints, $props) {
+      max-width: map.get($custom-breakpoints, $props);
+    } @else {
+      @include u-maxw($props);
+    }
```

Comment:
@mejiaj not sure if nested conditional statements are ideal here. Let me know if you have suggestions to clean this up!

Reply:
No issues. We can revisit if we run into anything.